sdk-config.json

```diff
@@ -6,6 +6,7 @@
     "authConfig_comment2": "Optional Full paths to Infinity server OAuth endpoints (Only needed if using custom OAuth service)",
     "authConfig_comment3": "Optional attributes are: authorize, token, revoke, authService, silentTimeout",
     "authService": "pega",
+    "iframeLoginUI": false,
```

Comment:
Suggest not having it in config file by default and document that someone could add it...but only if they are working with an identity provider login screen that will both display well in an iFrame as well as allow submitting in the iframe.